layouts/millennium_dstu2_sidebar.html

```diff
@@ -1,4 +1,7 @@
 <div id="js-sidebar" class="sidebar-shell">
+  <div class="expand-all">
```

Comment:
slightly tripping me up to use the class expand-all which is only true really half the time

how about expand-collapse ?

Reply:
Updated here: 3865c35e7b4aa380d934652b445386264bf801b1